doc/man1/flux-kvs.rst

```diff
@@ -48,18 +48,18 @@ arguments are described below.
 COMMANDS
 ========
 
-**namespace create** [-o owner] *name* [*name…​*]
+**namespace create** [-o owner] *name* [*name* ...]
    Create a new kvs namespace. User may specify an alternate userid of a
    user that owns the namespace via *-o*. Specifying an alternate owner
    would allow a non-instance owner to read/write to a namespace.
 
-**namespace remove** *name* [*name…​*]
+**namespace remove** *name* [*name...*]
```

Comment:
You are missing a space before the "...".

Reply:
Thanks! This PR was only removing some non-ascii characters that crept in due to auto-conversion from asciidoc to reStructuredText, and as such preserves the previous formatting I believe. If you have time, please open an issue on manpage formatting errors (I'm sure there are plenty!). We also accept pull requests! :wink: